src/index.html

```diff
@@ -22,6 +30,367 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__icon">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="Moyo Icon"
+            width="40"
+            height="40"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              class="header__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#smartphones"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            data-qa="hover"
+            class="header__item"
+          >
+            <a
+              data-qa="nav-hover"
+              class="header__link"
+              href="#laptops-computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
```

Comment:
The img tag is missing the required 'alt' attribute. This attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader). It's a good practice to always include an alt attribute for images. The value of the alt attribute should describe the image.